packages/flutter/README.md

```diff
@@ -7,110 +7,3 @@ focused on low-latency input and high frame rates on Android and iOS.
-
-```dart
-Localizations.of<MaterialLocalizations>(context, MaterialLocalizations);
-```
-
-This particular `Localizations.of()` expression is used frequently, so the MaterialLocalizations class provides a convenient shortand:
-
-```dart
-static MaterialLocalizations of(BuildContext context) {
-  return Localizations.of<MaterialLocalizations>(context, MaterialLocalizations);
-}
-
-/// References to the localized values defined by MaterialLocalizations
-/// are typically written like this:
-
-tooltip: MaterialLocalizations.of(context).backButtonTooltip,
-```
-
-The example that follows can be viewed as a template for using Flutter's localizations classes. It defines a single class, MyLocalizations, that contains the app's localized resources (just one for the example). It uses the initializeMessages() function generated by the Dart intl package to load resources and Intl.message() to look them up.
-
-```dart
-class MyLocalizations {
-  MyLocalizations(this.locale);
-
-  final Locale locale;
-
-  static Future<MyLocalizations> load(Locale locale) {
-    return initializeMessages(locale.toString())
-      .then((Null _) {
-        return new MyLocalizations(locale);
-      });
-  }
-
-  static MyLocalizations of(BuildContext context) {
-    return Localizations.of<MyLocalizations>(context, MyLocalizations);
-  }
-
-  String title() => Intl.message('<title>', name: 'title', locale: locale.toString());
-  // ... more Intl.message() methods like title()
-}
-```
-
-A class based on the `intl` package imports a generated message catalog that provides
-the `initializeMessages()` function and the per-locale backing store for `Intl.message()`.
-The message catalog is produced by an `intl` tool that analyzes the source code for
-classes that contain `Intl.message()` calls. In this case that would just be the
-`MyLocalizations` class.
-
-An instance of `MyLocalizations` is created by a simple LocalizationsDelegate:
-
-```dart
-class MyLocalizationsDelegate extends LocalizationsDelegate<MyStrings> {
-  @override
-  Future<MyStrings> load(Locale locale) => MyStrings.load(locale);
-
-  @override
-  bool shouldReload(MyLocalizationsDelegate old) => false;
-}
-```
-
-An instance of `MyLocalizationsDelegate` is provided to application's MaterialApp (or its WidgetsApp for non-material applications):
-
-```dart
-class MyApp extends StatelessWidget {
-  @override
-  Widget build(BuildContext context) {
-    return new MaterialApp(
-      localizationsDelegates: <LocalizationsDelegate>[
-        new MyLocalizationsDelegate()
-      ],
-      supportedLocales: <Locale>[
-        const Locale('en', 'US'), // US English
-        const Locale('fr', 'CA'), // Candian French
-      ],
-      child: home: new Scaffold(
-        appBar: new AppBar(
-          title: new Text(MyLocalizations.of(context).title),
```

Comment:
Note that in the example for MyLocalizations above, the title member is a function, not a getter.  One of them is probably wrong... :-)

Reply:
Yes, thanks!